projects/plugins/social/src/class-meta-tags.php

```diff
@@ -0,0 +1,223 @@
+<?php
+/**
+ * Adds meta tags to pages that need it.
+ *
+ * @package automattic/jetpack-social-plugin
+ */
+
+namespace Automattic\Jetpack\Social;
+
+/**
+ * Adds the meta tags.
+ */
+class Meta_Tags {
+	/**
+	 * This list is copied verbatim from class.jetpack.php
+	 *
+	 * Note: All in One SEO Pack, All in one SEO Pack Pro, WordPress SEO by Yoast, and WordPress SEO Premium by Yoast automatically deactivate
+	 * Jetpack's Open Graph tags via filter when their Social Meta modules are active.
```

Comment:
Will this be true for us too? I don't know if we're using the same filters that they use to deactivate it.

Reply:
Yeah, I've checked and they both use the `jetpack_enable_open_graph` filter, which is the same one we use here.